releases/release-5.3.2.md

```diff
@@ -0,0 +1,177 @@
+    (dup: release-5.4.0.md > Bug fixes> TiDB)- Fix the issue of potential data index inconsistency in optimistic transaction mode [#30410](https://github.com/pingcap/tidb/issues/30410)
+    (dup: release-5.2.4.md > Bug fixes> TiDB)- Fix the issue that a SQL operation is canceled when its JSON type column joins its `CHAR` type column [#29401](https://github.com/pingcap/tidb/issues/29401)
+    - In the event that a network connectivity event occurred, TiDB did not always free the resources of the disconnected session correctly. This has been fixed to ensure that open transactions are rolled back, as well as other associated resources. [#34722](https://github.com/pingcap/tidb/issues/34722)
+    - Fixed an issue where extra datums may break binlog. [#33608](https://github.com/pingcap/tidb/issues/33608)
+    (dup: release-6.1.0.md > Bug fixes> TiDB)- Fix the issue that query result might be wrong when Plan Cache is started in the RC isolation level [#34447](https://github.com/pingcap/tidb/issues/34447)
+    (dup: release-5.4.1.md > Bug Fixes> TiDB)- Fix the session panic that occurs when executing the prepared statement after table schema change with the MySQL binary protocol [#33509](https://github.com/pingcap/tidb/issues/33509)
+    (dup: release-5.4.1.md > Bug Fixes> TiDB)- Fix the issue that the table attributes are not indexed when a new partition is added and the issue that the table range information is not updated when the partition changes [#33929](https://github.com/pingcap/tidb/issues/33929)
+    (dup: release-5.4.1.md > Bug Fixes> TiDB)- Fix the issue that the TiDB server might run out of memory when the `INFORMATION_SCHEMA.CLUSTER_SLOW_QUERY` table is queried. This issue can be triggered when you check slow queries on the Grafana dashboard [#33893](https://github.com/pingcap/tidb/issues/33893)
+    (dup: release-6.1.0.md > Bug fixes> TiDB)- Fix the issue that some DDL statements might be stuck for a period after the PD node of a cluster is replaced [#33908](https://github.com/pingcap/tidb/issues/33908)
+    (dup: release-5.4.1.md > Bug Fixes> TiDB)- Fix the issue that granting the `all` privilege might fail in clusters that are upgraded from v4.0 [#33588](https://github.com/pingcap/tidb/issues/33588)
+    (dup: release-5.4.1.md > Bug Fixes> TiDB)- Fix wrong results of deleting data of multiple tables using `left join` [#31321](https://github.com/pingcap/tidb/issues/31321)
+    (dup: release-5.4.1.md > Bug Fixes> TiDB)- Fix a bug that TiDB may dispatch duplicate tasks to TiFlash [#32814](https://github.com/pingcap/tidb/issues/32814)
+    (dup: release-5.2.4.md > Bug fixes> TiDB)- Fix the issue that the background HTTP service of TiDB might not exit successfully and makes the cluster in an abnormal state [#30571](https://github.com/pingcap/tidb/issues/30571)
```

Comment:
- Fix the problem that tidb-server may encounter crash because of `fatal error: concurrent map read and map write` [#35340](https://github.com/pingcap/tidb/issues/35340)